openwisp_monitoring/monitoring/migrations/0017_merge_traffic_metrics_and_fill_metric_config.py

```diff
@@ -13,8 +13,8 @@
 
 
 def merge_traffic_metrics(apps, schema_editor):
-    Metric = load_model('monitoring', 'Metric')
-    Chart = load_model('monitoring', 'Chart')
+    Metric = apps.get_model('monitoring', 'Metric')
+    Chart = apps.get_model('monitoring', 'Chart')
```

Comment:
Using `apps.get_model` is always preferred in Migrations, don't know how I missed it :sweat_smile: (maybe due to #161, but it's resolved now so no worries). Thanks!